src/main/java/lessons/hw5/model/User.java

```diff
@@ -0,0 +1,36 @@
+package lessons.hw5.model;
+
+import java.util.List;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.Table;
+import lombok.Data;
+
+@Data
+@Entity
+@Table(name = "users")
+public class User {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long id;
+  private String email;
+  private String name;
+
+  @ManyToMany
+  @JoinTable(
+    name = "user_role", // table name for relation
+    joinColumns = @JoinColumn(name = "id_user"),
+    inverseJoinColumns = @JoinColumn(name = "id_role")
+  )
+  private List<Role> roles;
```

Comment:
Optional improvement: could be Set
```suggestion
  private Set<Role> roles;
```

Reply:
ok